app/routes/dashboards/dashboard.js

```diff
@@ -2035,7 +2035,11 @@ export default Ember.Route.extend({
 
                     let parameter_value;
                     if (param.parameterName in transition.queryParams) {
-                        parameter_value = transition.queryParams[param.parameterName];
+                        if (param.parameterName == "page") {
+                            parameter_value = (Number(transition.queryParams[param.parameterName])*10)-10;
+                        } else {
+                            parameter_value = transition.queryParams[param.parameterName];
+                        }
```

Comment:
Yeah, and the entire dashboard refreshes when ideally it should just be the results list.